app/Http/Controllers/Eportalcontroller.php

```diff
@@ -132,11 +144,21 @@ public function edit($id)
      * @param  int  $id
      * @return \Illuminate\Http\Response
      */
-    public function update(Request $request, $id)
-    {
-        //
+    public function update(Request $request)
+    {   
+        $pdirectory = Directorie::find($request->pdirectory); 
+        DB::table("directories")->where('id',$request->id)->update(["file_name"=>$request->rename]);
```

Comment:
let's use Laravel eloquent more for any business logic. That provides a cleaner and more secure way to make insertions/additions